src/main/java/slimeknights/tconstruct/library/book/content/ContentTool.java

```diff
@@ -64,8 +64,8 @@ public void load() {
       toolName = parent.name;
     }
     if(tool == null) {
-      tool = TinkerRegistry.getTools().stream()
-                           .filter(toolCore -> toolName.equals(toolCore.getIdentifier()))
+      tool = (ToolCore) TinkerRegistry.getTools().stream()
+                           .filter(toolCore -> toolName.equals(((ToolCore) toolCore).getIdentifier()))
```

Comment:
Neither of the book related issues caused a crash, I checked thoroughly because I expected one there too.